nixos/lib/test-driver/test-driver.py

```diff
@@ -0,0 +1,680 @@
+#! /somewhere/python3
+
+from __future__ import print_function
```

Comment:
This shouldn't be needed on Python 3 - we're already in the future.

Reply:
Thx for your review! I won't be able to do any of this today, but i will incorporate soon!